bin/pygrb/pygrb_make_summary_page

```diff
@@ -0,0 +1,1034 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2011 Ian W. Harry
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# =============================================================================
+# Preamble
+# =============================================================================
+
+from __future__ import division
+
+import os,sys,datetime,re,glob,shutil,ConfigParser
+from argparse import ArgumentParser
+import random
+import string
+from glue import markup
+from pylal import grbsummary,rate,antenna,git_version
+from lal.gpstime import gps_to_utc,LIGOTimeGPS
+
+__author__  = "Ian Harry <ian.harry@astro.cf.ac.uk>"
+__version__ = "git id %s" % git_version.id
+__date__    = git_version.date
+
+
+def make_external_call(\
```

Comment:
Have changed to existing function, but have commented these calls out as there is probably a better way to accomplish what we want here and they would only run successfully with shell=True, which is a possible vulnerability according to the documentation of make_external_call.